package/base-files/files/etc/init.d/led

```diff
@@ -58,11 +72,15 @@ load_led() {
 		[ "$default" = 0 ] &&
 			echo 0 >/sys/class/leds/${sysfs}/brightness
 
+		[ $sysfs_brightness = 0 ] && {
```

Comment:
Is it required to switch the trigger just because the brightness is 0? We don't know if there's external code/users expecting certain trigger values regardless of the current effective brightness.

If the LED stops shining on brightness 0, regardless of the trigger I'd say you can remove this extra handling.

Reply:
Yes, sadly. It's an edge case that I have experienced.
If you set the brightness to 0, the led is considered off by the system. When a led is off, even if you change the trigger, the properties of the trigger won't become available in the sysfs. 
If we then try to configure one of those properties, it will fail with errors.